### ISSUE TITLE
decide on naming convention

### ISSUE THREAD

**Issue by vks** [vks/special-fun#1]
Currently only a few of the special functions are exposed to Rust, with names that differ from CEPHES. The names used by CEPHES are extremely short, so we probably want to use longer names.

**Comment by tmarkovich** [comment]
I'm happy to follow the `scipy` naming conventions if you agree that those make sense.

**Comment by vks** [comment]
Yes, that makes sense. I think [some of them](https://docs.scipy.org/doc/scipy/reference/special.html) are very short, but given scipy's popularity it is probably a good idea to use the same names.
